src/ExpressionEvaluator/VisualBasic/Test/ExpressionCompiler/PseudoVariableTests.vb

```diff
@@ -365,62 +365,6 @@ End Class"
                 End Sub)
         End Sub
 
-        ''' <summary>
-        ''' The assembly-qualified type name may be from an
-        ''' unrecognized assembly. For instance, if the type was
-        ''' defined in a previous evaluation, say an anonymous
-        ''' type (e.g.: evaluate "o" after "o = New With { .P = 1 }").
-        ''' </summary>
-        <WorkItem(1449, "https://github.com/dotnet/roslyn/issues/1449")>
-        <Fact(Skip:="https://github.com/dotnet/roslyn/issues/1449")>
```

Comment:
Removed test since it no longer represented the issue. (The assembly from the previous evaluation is available in the debugger.)